.github/workflows/release_tools.yaml

```diff
@@ -0,0 +1,113 @@
+        timeout-minutes: 60
+
+        runs-on: ubuntu-latest
+
+        env:
+            DEBIAN_FRONTEND: noninteractive
+            JAVA_HOME: /usr/lib/jvm/java-8-openjdk-amd64/
+
+        container:
+            image: connectedhomeip/chip-build-android:0.4.33
+            volumes:
+                - "/tmp/log_output:/tmp/test_logs"
+                - "/tmp/output_binaries:/tmp/output_binaries"
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  ref: "${{ github.event.inputs.commit }}"
+                  submodules: true
+            - name: Bootstrap
+              timeout-minutes: 10
+              run: scripts/build/gn_bootstrap.sh
+            - name: Install Python CHIP Controller dependencies
+              timeout-minutes: 10
+              run: |
+                  echo "deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports $(lsb_release -sc) main restricted" > /etc/apt/sources.list.d/arm64.list
+                  echo "deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports $(lsb_release -sc)-updates main restricted" >> /etc/apt/sources.list.d/arm64.list
+                  apt update
+                  apt install -y --no-install-recommends g++-aarch64-linux-gnu libgirepository1.0-dev
+                  dpkg --add-architecture arm64
+                  apt install -y --no-install-recommends libavahi-client-dev:arm64 libglib2.0-dev:arm64 libssl-dev:arm64
+            - name: Build x64 Python CHIP Controller
+              timeout-minutes: 10
+              run: |
+                  scripts/run_in_build_env.sh "gn gen out/python_x64 --args='chip_mdns=\"platform\"'"
+                  scripts/run_in_build_env.sh "ninja -C out/python_x64 python"
+                  cp out/python_x64/controller/python/chip-*.whl /tmp/output_binaries/
+            - name: Build arm64 Python CHIP Controller
+              timeout-minutes: 10
+              run: |
+                  scripts/run_in_build_env.sh "gn gen out/python_arm64 --args='chip_mdns=\"platform\"
+                      custom_toolchain=\"//build/toolchain/custom\"
+                      target_cc=\"aarch64-linux-gnu-gcc\"
+                      target_cxx=\"aarch64-linux-gnu-g++\"
+                      target_ar=\"aarch64-linux-gnu-ar\"
+                      target_cpu=\"arm64\"'"
+                  scripts/run_in_build_env.sh "ninja -C out/python_arm64 python"
+                  cp out/python_arm64/controller/python/chip-*.whl /tmp/output_binaries/
+            - name: Build arm CHIPTool
+              timeout-minutes: 10
+              env:
+                TARGET_CPU: arm
+              run: |
+                  scripts/examples/android_app.sh
+                  yes | "$ANDROID_HOME/tools/bin/sdkmanager" --licenses
+                  cd src/android/CHIPTool
+                  ./gradlew build
+                  cp app/build/outputs/apk/debug/app-debug.apk /tmp/output_binaries/chiptool-arm.apk
```

Comment:
What about to figure out a little bit more meaningful name e.g.: `chip-tool-android-arm` or sth similar?

Reply:
Fine with me. I thought `android` was implied from `apk`, but I can make it more explicit.